speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/rppUtils/Random.hpp

```diff
@@ -5,20 +5,35 @@
 #define BYTE   uint8_t
 #define USHORT uint16_t
 
+#define SHORT_MAXIMUM_VALUE 65535
+#define SHORT_MAX_VALUE     65536
```

Comment:
Any reason to not use ` std::numeric_limits::max()`?

Reply:
This was refactored by @KungFuDonkey, I will replace it with `USHRT_MAX`

Reply:
no reason to use a function where a constant can used